credential/util/util.go

```diff
@@ -7,59 +7,93 @@ import (
 	"github.com/TBD54566975/ssi-sdk/credential"
 	"github.com/TBD54566975/ssi-sdk/credential/signing"
 	"github.com/goccy/go-json"
+	"github.com/lestrrat-go/jwx/v2/jwt"
 	"github.com/pkg/errors"
 )
 
-// CredentialsFromInterface turn a generic cred into a known shape without maintaining the proof/signature wrapper
-func CredentialsFromInterface(genericCred any) (*credential.VerifiableCredential, error) {
+// ToCredential turn a generic cred into its known object model
+func ToCredential(genericCred any) (*credential.VerifiableCredential, error) {
```

Comment:
@nitro-neal refactored this method and the one below to handle jwt/json properly to fix #165